tools/Makefile

```
@@ -77,4 +77,5 @@ update_cache_churn: $(CHURN_WEEK_FILE)
 	$(MAKE) churn $(CHURN_WEEK_FILE)
 
 $(CHURN_WEEK_FILE):
+	mkdir -p $$(dirname $(CHURN_WEEK_FILE))
```

You can avoid a shell by using [`dir`](https://www.gnu.org/software/make/manual/html_node/File-Name-Functions.html#File-Name-Functions) :)
```suggestion
	mkdir -p $(dir $(CHURN_WEEK_FILE))
```